### DISCUSSION TITLE
./prowler is a directory

### DISCUSSION THREAD

**Discussion by RahulAhujatoo** [prowler-cloud/prowler#2603] [Q&A] [answered]
I tried to run prowler on AWS shell but it isn't working.
After git clone I ran "cd prowler" and then "./prowler -l" but it says
"./prowler is a directory"

what should i do?

**Comment by jfagoagas** [comment, marked as answer]
Hi @RahulAhujatoo, to run Prowler after a `git clone` you must follow the `Github` section in the ProwlerPro documentation QuickStart at https://docs.prowler.cloud/en/latest/#installation.

Please check it out and let us know if something is not working for you.

Also, in Prowler v3 the main file is under the root path --> `./prowler.py`.

Thanks for using Prowler 🚀
